README.rst

```diff
@@ -85,6 +85,8 @@ Command-line options
     Optional process count limit for `rsync` processes. By default there's no limit so rsyncing will be in parallel
     for all test nodes.
 
+* `--cloud-rsync-cipher`
+    Optional ssh cipher selection for `rsync` processes. aes128-gcm@openssh.com by default.
```

Comment:
please add a reason for choosing the non-default cipher

Reply:
Added more verbose description.